app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "AllAboutOlaf",
+  "displayName": "All About Olaf"
+}
```

Comment:
I had to do some digging. I think it's part of a CLI to more easily recreate native folders? https://github.com/facebook/react-native/pull/12162